sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -86,6 +86,7 @@ class SparkSession private(
 
   private[sql] def this(sc: SparkContext) {
```

Comment:
@RussellSpitzer, mind if I ask to note that this is currently not used in Scala code base but PySpark and tests in Scala side?